### ISSUE TITLE
Flashing latest LSDJ versions fails

### ISSUE THREAD

**Issue by naught101** [gheja/ems-flasher#6]
When I flash the latest versions of LSDJ using ems-flasher, it appears to work, but then when I try to load on a Gameboy Pocket, it fails. Here is some command line output:
```
$ ./ems-flasher-0.03/ems-flasher --verbose --write lsdj.gb --bank 1
trying to find EMS cart
claimed EMS cart
base address is 0x0
Writing ROM file lsdj.gb
Successfully wrote 1048576 from lsdj.gb
$ ./ems-flasher-0.03/ems-flasher --title --verbose             
trying to find EMS cart
claimed EMS cart
base address is 0x0
Bank 0: LSDj-v5.3.5�
Hardware support: DMG <+CGB>
Cartridge header checksum invalid. This game will NOT boot on real hardware.
1024 KB ROM
Bank 1: LSDj-v5.3.5�
Hardware support: DMG
Cartridge header checksum invalid. This game will NOT boot on real hardware.
1024 KB ROM
```

I'm not sure what's going wrong. I don't yet have any other hardware to try this on. Is there anything else I can try, or info I can provide?

**Comment by superjamie** [comment]
Should be resolved by: https://github.com/gheja/ems-flasher/pull/7

**Comment by superjamie** [comment]
This is just a cosmetic issue in `ems-flasher` btw. As long as the actual ROM header checksum is correct then the console will run the ROM. You can also confirm the ROM checksums with `rgbfix` from RGBDS.

**Comment by gheja** [comment]
I merged #7, so I am closing this Issue.
